packages/stream_chat_flutter/lib/src/message_input/translucent_scafold.dart

```diff
@@ -0,0 +1,101 @@
+import 'package:flutter/material.dart';
+
+class TranslucentScaffold extends StatelessWidget {
+  const TranslucentScaffold({
+    super.key,
+    required this.body,
+    this.appBar,
+    this.bottomNavigationBar,
+  });
+
+  final Widget? body;
+  final Widget? appBar;
+  final Widget? bottomNavigationBar;
+
+  @override
+  Widget build(BuildContext context) {
+    const background = Color(0xff0A070B);
```

Comment:
Theres no need to include TranslucentScaffold to the SDK. Please remove it.

Reply:
Will remove once we integrate in the chatSDK, using for now for reference and complete look and feel of app.